backend/estimates/src/main/resources/db/migrate.sh

```
@@ -1,3 +1,11 @@
 #!/bin/sh
-
-flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true   -outOfOrder=true migrate
+baseurl=$DB_URL
+echo "the baseurl : $DB_URL"
+schemasetter="?currentSchema="
+schemas=$SCHEMA_NAME
+echo "the schemas : $schemas"
+for schemaname in ${schemas//,/ }
```

Shellcheck (0.10.0)

[warning] In POSIX sh, string replacement is undefined.

(SC3060)